src/containers/DailyReportPage/index.js

```diff
@@ -15,8 +15,8 @@ const DailyReportPage = ({ match }) => (
     </LeftContent>
     <RightContent>
       <Switch>
-        <Route exact path={match.url} component={DailyReportContainer} />
-        <Route path={`${match.path}/new`} component={CreateDailyReportContainer} />
+        <Route exact path={match.url} render={props => <DailyReportContainer {...props} />} />
+        <Route path={`${match.path}/new`} render={props => <CreateDailyReportContainer {...props} />}/>
```

Comment:
By using `history` in `utils`, we don't need to pass `...props` to the component like this. 

Should be the same as before.
```
<Route exact path={match.url} component={DailyReportContainer} />
```

Reply:
@phuongthuan 
Thanks, however, I actually needed to pass `location` by `react-router` to access query string from children components in `DailyReportContainer`.

Like, 
location.seacrh
=> `page?=1`

So I'm gonna leave `DailyReportContainer` as it is and fix `CreateDailyReportContainer`.